pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/mercury/MercuryOrderDao.java

```diff
@@ -137,6 +134,7 @@ public static void updateOrderStatus(BaseMercuryStatusMessage baseMercuryStatusM
     }
 
     public Map<String, MercuryOrderDto> getPossibleBarcodesForParticipant(String ddpParticipantId) {
+        String errorMessage = "Error getting eligible barcodes for a mercury order for participant %s";
```

Comment:
Sorry, I was not clear. You can add the ddpParticipantId right here.

Reply:
Oh sorry I misunderstood!